### ISSUE TITLE
Vanilla doors have missing texture (pink and black) on bottom half, top half invisible

### ISSUE THREAD

**Issue by darthvader45** [Ordinastie/MalisisDoors#340]
For some reason this happens to naturally generated doors (ex. doors in villages, strongholds, etc.).  I figured this was an Optifine issue, but I've run the same version of Optifine on other versions and have not had this issue.  I then assumed it was the 6.5.1 version of MalisisCore, which apparently is considered a snapshot.  Perhaps that is the issue?

**Comment by WolfMad** [comment]
in the malisisdoors.cfg config file setting B:config.modifyVanillaDoors to false fixes the problem

**Comment by darthvader45** [comment]
Yeah, though that messes with player-placed doors, which I don't want to do.  Hopefully there can be a fix, though this is indeed a workaround.
